docs/UserGuide.md

```diff
@@ -357,9 +371,54 @@ If your changes to the data file makes its format invalid, the data will be disc
 
 
 --------------------------------------------------------------------------------------------------------------------
-
+#Summary of Commands
+##Contacts Tab
+
+| Command | Intention                           | Format                                                                                         |
+|---------|-------------------------------------|------------------------------------------------------------------------------------------------|
+| add     | add a new person                    | `add n/NAME [p/PHONE] [e/EMAIL] [d/{SOCIAL MEDIA PLATFORM}, {ID}]... [t/TAG]`                  |
+| add     | add a tag to an existing person     | `add NAME t/NEWTAG` or `add INDEX t/NEWTAG`                                                    |
+| add     | add a new social media to a person  | `add NAME d/{SOCIAL MEDIA PLATFORM}, {ID}` or `add INDEX d/{SOCIAL MEDIA PLATFORM}, {ID}`      |
+| find    | search for a person                 | `[find] KEYWORD [MORE_KEYWORDS]`                                                               |
+| view    | view the details of a person        | `view INDEX` or click the Contact/Meeting under the list.                                      |
+| edit    | edit details of a person            | `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [d/{SOCIAL MEDIA PLATFORM}, {ID}]… [t/TAG]…`          |
+| copy    | copy all details of a person        | `copy NAME`                                                                                    |
+| copy    | copy one detail of a person         | perform the `view` command and click on the relevant label values to copy                      |
+| delete  | delete a person                     | `delete NAME` or `delete INDEX`                                                                |
+| delete  | delete a tag from a person          | `delete NAME t/TAG` `delete INDEX t/TAG`                                                       |
+| delete  | delete a social media from a person | `dlete NAME d/{SOCIAL MEDIA PLATFORM}, {ID}` or `delete INDEX d/{SOCIAL MEDIA PLATFORM}, {ID}` |
+
+##Meetings Tab
```

Comment:
Same as @zihaowrez mentioned, all ## values require a space after for the correct formatting.